src/client/v2/algod/sendRawTransaction.ts

```diff
@@ -49,7 +50,17 @@ export default class SendRawTransaction extends JSONRequest {
 
   async do(headers = {}) {
     const txHeaders = setSendTransactionHeaders(headers);
-    const res = await this.c.post(this.path(), this.txnBytesToPost, txHeaders);
-    return res.body;
+    const res = await this.c.post(
+      this.path(),
+      Buffer.from(this.txnBytesToPost),
```

Comment:
I think `Buffer.from` conversions are unnecessary now:
```suggestion
      this.txnBytesToPost,
```